_docs/_latest/use/inventory.md

```diff
@@ -6,12 +6,14 @@ order: 002
 # {{ page.title }}
 
 Forseti Inventory collects and stores information about your Google Cloud Platform (GCP)
-resources. Forseti Scanner and Enforcer use Inventory data to perform operations.
+resources. The Inventory data is transformed to data model, which will be used
+by Forseti Scanner and Explainer to perform operations.
 
 ## Running Inventory
 
-Before you start using Inventory, you'll need to make sure you have the  
-[proper permission setup]({% link _docs/latest/configure/gsuite.md %}) 
```

Comment:
Do we not want to mention G Suite DwD steps here?

Reply:
done